scripts/ccpp_capgen.py

```diff
@@ -520,11 +533,13 @@ def parse_scheme_files(scheme_filenames, run_env):
     table_dict = {} # Duplicate check and for dependencies processing
     header_dict = {} # To check for duplicates
     known_ddts = list()
+    # end if
```

Comment:
?

Reply:
orphan put out of his misery.

(removed)